src/cmd/prepare.go

```diff
@@ -20,11 +21,11 @@ var prepareCmd = &cobra.Command{
 }
 
 var prepareTransformGitLinks = &cobra.Command{
-	Use:   "patch-git [HOST] [FILE]",
+	Use:     "patch-git [HOST] [FILE]",
```

Comment:
was this whitespace intentional, linting maybe?

Reply:
Yes the linter did that (same with the other whitespace changes).  I included it in my commit since this file may have not been touched by the linter yet.  I think my linter settings are correct, but can work to correct it if not.

Reply:
Played around with just running `pre-commit run -a` and on my machine it adds spaces to a lot of files...  Should we do a file cleanup PR to bring everything into standards with linting?

Reply:
(there are also a few more rules we may have to disable like `gochecknoglobals` as that is flagging on these config options for me)

Reply:
(discussed this offline, proceeding)